### PR TITLE
Additional metrics implementation

### PR DESCRIPTION
**Pull request by jonathanschirp**
Configuation of additional metrics for multiclass and multilabel experiments.

### CODE REVIEW

**Comment by jonathanschirp** [comment]
Implemented AUPR, AUORC and ECE metrics for Multiclass experiments. The metrics are enabled by default when using module/multiclass.yaml

File: birdset/modules/metrics/multiclass.py [hunk 1]
```diff
@@ -30,10 +31,25 @@ def __init__(
         self.val_metric_best: torchmetrics.Metric =torchmetrics.MaxMetric()
      
         self.add_metrics: torchmetrics.MetricCollection = torchmetrics.MetricCollection({
-            'F1': torchmetrics.F1Score(
+            'F2': torchmetrics.F1Score(
```
Comment (raphaelschwinger) [inline birdset/modules/metrics/multiclass.py:34]:
```suggestion
            'F1': torchmetrics.F1Score(
```

File: birdset/modules/multiclass_module.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import torch
```
Comment (raphaelschwinger) [inline birdset/modules/multiclass_module.py:1]: @jonathanschirp This is not needed. The `base_module` is for multiclass and we have the `multilabel_module` for multilabel.

File: configs/datamodule/transforms/bird_default_multiclass.yaml [hunk 3]
```diff
@@ -3,7 +3,6 @@ sampling_rate: ${module.network.sampling_rate}
 model_type: ${module.network.model_type}
 task: ${datamodule.dataset.task}
 max_length: 5
-n_classes: ${datamodule.dataset.n_classes}
```
Comment (raphaelschwinger) [inline configs/datamodule/transforms/bird_default_multiclass.yaml:6]: Good catch!

**Review by raphaelschwinger — CHANGES_REQUESTED** [review]
@jonathanschirp Thanks for your PR. Good work!
Sorry it took so long to review. Took some effort to get our repo up to date!

I tested it successfully with experiment `local/esc50/efficientnet`!
Pls fix the small suggestion down below and then this gets merged 🔥

**Comment by jonathanschirp** [comment]
Should be ready to merge? Should I do that myself or do you want to @raphaelschwinger?

**Comment by raphaelschwinger** [comment]
@jonathanschirp Thanks! First PR done 🙌  Best practice is that the reviewer merges a PR, i will do that now!

**Review by raphaelschwinger — APPROVED** [review]

**Pull request merged**